src/UWP/ArcGISRuntime.UWP.Viewer/Samples/GeometryEngine/Buffer/Buffer.xaml

```diff
@@ -0,0 +1,19 @@
+﻿<UserControl
+    x:Class="ArcGISRuntime.UWP.Samples.Buffer.Buffer"
+    x:Name="_this"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:ArcGISRuntime.UWP.Samples.Buffer"
+    xmlns:esriUI="using:Esri.ArcGISRuntime.UI.Controls">
+    <Grid>
+        <Grid.RowDefinitions>
+            <RowDefinition Height="auto" />
+            <RowDefinition Height="*" />
+        </Grid.RowDefinitions>
+        <StackPanel  Orientation="Horizontal" Grid.Row="0">
```

Comment:
Per the new style convention, row and column need to be set before other properties (but after the name).